packages/vue-query/src/__tests__/useMutation.test.ts

```diff
@@ -35,11 +35,8 @@ describe('useMutation', () => {
 
   test('should return error when request fails', async () => {
     const mutation = useMutation(errorMutator)
-
-    mutation.mutate()
```

Comment:
I do not think changes in this file are relevant

Reply:
Ok will take another look - for some reason or another passing no args was failing the tests.

Reply:
<img width="1116" alt="Screen Shot 2022-10-22 at 4 33 44 PM" src="https://user-images.githubusercontent.com/25216513/197361369-039678aa-7f9d-4eaf-a0c5-2cca5319ed1d.png">

Any idea why these changes would cause something as such?

Reply:
Actually no idea.
Does it work on `main` branch?
What TS version do you have installed?

Reply:
that test is running on your test suite - not my local so not sure if my ts version is relevant

Reply:
export type MutateFunction<
  TData = unknown,
  TError = unknown,
  TVariables = void,
  TContext = unknown,
> = (
  variables: TVariables,
  options?: MutateOptions<TData, TError, TVariables, TContext>,
) => Promise<TData> 

it seems completely reasonable to get an error for not passing args - variables is not optional. Maybe I somehow removed a Partial<MutateFunction> somewhere... pretty strange